frontend/src/metabase/admin/tasks/components/Logs/Logs.unit.spec.tsx

```diff
@@ -97,6 +97,20 @@ describe("Logs", () => {
         expect(screen.getByText(errMsg)).toBeInTheDocument();
       });
     });
+
+    it("should stop polling on unmount", async () => {
+      fetchMock.get("path:/api/util/logs", [log]);
+      const { unmount } = render(<Logs />);
+      expect(
+        await screen.findByText(new RegExp(log.process_uuid)),
+      ).toBeInTheDocument();
+
+      unmount();
+      act(() => {
+        jest.advanceTimersByTime(1100); // wait longer than polling period
+      });
+      expect(utilSpy).toHaveBeenCalledTimes(1); // should not have been called
```

Comment:
Nit:
This comment gave me a pause and sent me on a 15 minute hunt until I realized what you probably wanted to say.
Should not have been called **again**?

Probably best to remove the comment itself. The code should be self explanatory.
```suggestion
      expect(utilSpy).toHaveBeenCalledTimes(1);
```

Reply:
I agree with you, it seems confusing. It's better to remove it